salt/states/debconfmod.py

```diff
@@ -205,23 +202,20 @@ def set(name, data, **kwargs):
         if args["type"] == "boolean":
             args["value"] = "true" if args["value"] else "false"
 
-        if (
-            current is not None
-            and [key, args["type"], six.text_type(args["value"])] in current
-        ):
+        if current is not None and [key, args["type"], str(args["value"])] in current:
             if ret["comment"] == "":
                 ret["comment"] = "Unchanged answers: "
-            ret["comment"] += ("{0} ").format(key)
+            ret["comment"] += ("{} ").format(key)
         else:
             if __opts__["test"]:
                 ret["result"] = None
-                ret["changes"][key] = ("New value: {0}").format(args["value"])
+                ret["changes"][key] = ("New value: {}").format(args["value"])
             else:
                 if __salt__["debconf.set"](name, key, args["type"], args["value"]):
                     if args["type"] == "password":
                         ret["changes"][key] = "(password hidden)"
                     else:
-                        ret["changes"][key] = ("{0}").format(args["value"])
+                        ret["changes"][key] = ("{}").format(args["value"])
```

Comment:
```suggestion
                        ret["changes"][key] = "{}".format(args["value"])
```
I'm tired of reviewing and point these useless `()` out. If you're not tired of fixing them, there are others on this module..